src/box/xrow.c

```diff
@@ -90,8 +97,22 @@ mp_encode_vclock_ignore0(char *data, const struct vclock *vclock)
 	return data;
 }
 
+static inline char *
+mp_encode_vclock_ignore0(char *data, const struct vclock *vclock)
+{
+	data = mp_encode_map(data, vclock_size_ignore0(vclock));
+	return mp_encode_vclock_impl(data, vclock, true);
+}
+
+static inline char *
+mp_encode_vclock(char *data, const struct vclock *vclock)
+{
+	data = mp_encode_map(data, vclock_size(vclock));
+	return mp_encode_vclock_impl(data, vclock, false);
+}
+
 static int
-mp_decode_vclock_ignore0(const char **data, struct vclock *vclock)
```

Comment:
It seems you don't use `mp_decode_vlcock` in this patch. Only the `mp_decode_vclock_ignore0`.
Let's leave only that version. So that noone decodes the vclock with 0th component accidentally

Reply:
I use it in subsequent commits in order to decode vclock inside `CURSOR`. There's also a comment about this in commit message

Reply:
Yes, indeed. I missed that.